src/FSharp.Data.GraphQL.Server/Schema.fs

```diff
@@ -107,10 +107,17 @@ type Schema<'Root> (query: ObjectDef<'Root>, ?mutation: ObjectDef<'Root>, ?subsc
         | _ -> failwithf "Unexpected value of typedef: %O" typedef
 
     let introspectInput (namedTypes: Map<string, IntrospectionTypeRef>) (inputDef: InputFieldDef) : IntrospectionInputVal =
-        { Name = inputDef.Name
-          Description = inputDef.Description
-          Type = introspectTypeRef false namedTypes inputDef.TypeDef
-          DefaultValue = inputDef.DefaultValue |> Option.map string }
+        match inputDef.DefaultValue with
+        | Some defaultValue ->
+            { Name = inputDef.Name
+              Description = inputDef.Description
+              Type = introspectTypeRef true namedTypes inputDef.TypeDef
```

Comment:
This can actually be written as `(Option.isSome inputDef.DefaultValue)` - in that case you won't need pattern match at all.